pyccel/codegen/wrapper/c_to_python_wrapper.py

```diff
@@ -80,10 +85,20 @@ def get_new_PyObject(self, name, is_temp = False):
         Variable
             The new variable.
         """
-        var = Variable(dtype=PyccelPyObject(),
-                        name=self.scope.get_new_name(name),
-                        memory_handling='alias',
-                        is_temp=is_temp)
+        if isinstance(dtype, CustomDataType):
+            try:
+                var = Variable(dtype=self._python_object_map[dtype],
+                                name=self.scope.get_new_name(name),
+                                memory_handling='alias',
+                                cls_base = self.scope.find(dtype.name, 'classes'),
+                                is_temp=is_temp)
+            except KeyError as e:
+                raise NotImplementedError("Can't return an object whose type was imported") from e
```

Comment:
What exactly does this message mean?

Reply:
It means that currently we cannot do:
File1.py
```python
class A:
    def __init__(self, x : int):
        self.x = x
```
File2.py
```python
from File1 import A

def f():
    return A(3)
```

The wrapper needs splitting into a .h and a .c file so that the class description is accessible from other modules, however this seemed beyond the scope of this PR so I left it as is for now

Reply:
Do we have an issue about this? If so, it could be referenced in the error message.

Reply:
I have created an issue and referenced it: 0f82d8d4b